src/main/java/core/basesyntax/service/AreaCalculator.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.model;
+
+public interface AreaCalculator {
+    String color = "";
```

Comment:
The `color` field should not be part of the `AreaCalculator` interface. Consider removing it, as interfaces should not contain fields that are not constants.